lessons/04-blast.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: lesson
+root: .
+title: Post-assembly - de novo RNA Transcriptome Analysis - handling Blast annotation output
+minutes: 5
+---
+
+## Learning Objectives 
+
+    In this lesson we will learn how to handle blast output file in order to arrive to the real homologs.
+
+    pre-request learning: How to blast online (ie NCBI) or by unix lesson.
+
+## Lesson 
+
+One of the interesting stages in de-novo RNA sequencing comes when you finally annotate your sequences.
+You can annotate your new contigs which theoretically represent your special organism transcriptome (all the RNA that is expressed in the cell) in different ways.
+In order to find homologs it is always recommended to work on the protein level, therefor we will first search for CDS in this transcript and then translate them into proteins. 
+One way to do so is to use the transdecode program which is included as a part of the Trinity package: http://trinityrnaseq.github.io/
+The logic behind this step is to filter all the contigs that have "junk" tails, for example one contig can be constructed from 40% CDS and 60% "junk" DNA. In that way we could latter reach  
+better results at the blast stage (It is also recommended to re-aligned the reads against these CDSs and not the original contigs in order to get real expression based on the CDS and not 

```

Comment:
re-aligned -> re-align. Also, I would pull the parenthetical sentence out into its own sentence or parenthetical expression. Otherwise make the leading _It_ lower case.